public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -453,6 +453,16 @@ export class GraphiteDatasource {
     return this.getFuncDefs();
   }
 
+  variableFormat(formatValue: Function) {
+    return (value: any, format: any, variable: any) => {
+      if (_.isArray(value) && value.length === 1) {
```

Comment:
could you move this into the default glob format that is in TemplateSrv? no need to have this special rule here it should be universal for all glob formats

Reply:
Phew! That's a great shout, I originally thought of doing that but decided to go with this first as `TemplateSrv` is used **all** over Grafana and wasn't entirely sure this something we wanted everywhere. Thanks for clearing that up - I'll move it there.

Reply:
Think only graphite is using the glob (default syntax) and think not including {} should be safe for single values (as it was previous behavior)